Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Footer.css';
+
+const Footer = () => {
+	return (
+		<footer>
+			<i class="far fa-comment"></i>
+			<ReShare />
+			<Like />			
+			<i class="far fa-envelope"></i>
+		</footer>
+	)
+}
+
+const ReShare = () => {
+	let num = 0;
+	return (		
+
+		<div className="share">
+			<i class="fas fa-sync-alt"></i>
+			<span>{num}</span>		
+		</div>
+	)
+}
+
+const Like = () => {
+	const heart = <i class="far fa-heart"></i>;
+	console.log(heart)
+	// heart.addEventListener('click', () => {
+	// 	return console.log("click works")
```

Comment:
your head is in the right place here, but obviously there is a different way to get this done with react. i'm sure it makes more sense after the lecture today.